src/js/views/patients/schedule/schedule_views.js

```diff
@@ -111,6 +114,51 @@ const SelectAllView = View.extend({
   },
 });
 
+const ListCountTemplate = hbs`
```

Comment:
can we get away with not using any of these views/templates and moving the worklist ones to a shared location?